assets/dev/js/frontend/handlers/base.js

```diff
@@ -197,7 +197,7 @@ module.exports = elementorModules.ViewModule.extend( {
 				settingsKeys = elementorFrontend.config.elements.keys[ type ] = [];
 
 				jQuery.each( settings.controls, ( name, control ) => {
-					if ( control.frontend_available ) {
+					if ( control.frontend_available || control.editor_available ) {
```

Comment:
Add the new property to this unset code: https://github.com/elementor/elementor/blob/6980d7cc46431f4554bf5b60fef181434f277cca/includes/base/controls-stack.php#L434C15-L451

Reply:
@KingYes Done